crates/uv-python/download-metadata.json

```diff
@@ -18,8 +18,8 @@
     "major": 3,
     "minor": 12,
     "patch": 3,
-    "url": "https://github.com/indygreg/python-build-standalone/releases/download/20240415/cpython-3.12.3%2B20240415-aarch64-unknown-linux-gnu-debug-full.tar.zst",
-    "sha256": "24daaf20123ac4b2b8657c1ac8227d391d2e5769d81237b68ee674569d307ad0"
+    "url": "https://github.com/indygreg/python-build-standalone/releases/download/20240415/cpython-3.12.3%2B20240415-aarch64-unknown-linux-gnu-lto-full.tar.zst",
+    "sha256": "a4f17d1e3b4ea0e4c2a3664f232c0857979522936af582f7de92b57050220f74"
```

Comment:
The archive for `lto` is more than 3 times the size of `debug`:

```
cpython-3.12.3+20240415-aarch64-unknown-linux-gnu-debug-full.tar.zst	50M
cpython-3.12.3+20240415-aarch64-unknown-linux-gnu-lto-full.tar.zst	187M
```

Looking at the docs, there's an [Install Only Archive](https://gregoryszorc.com/docs/python-build-standalone/main/distributions.html#install-only-archive). Does uv need the _full_ variant?

Reply:
I'm not sure why Rye opted not to use these (and our download preferences are based on the implementation over there). Here's an issue for discussion https://github.com/astral-sh/uv/issues/4834